train-system/src/test/java/hu/bme/mit/train/system/TrainSystemTest.java

```diff
@@ -52,6 +52,28 @@ public void OverridingJoystickPositionToNegative_SetsReferenceSpeedToZero() {
 
 	@Test
 	public void EmptyTest() {
-		
+
+	}
+
+	@Test
+	public void PeriodicSpeedChange() {
+		try {
```

Comment:
This is required in order not to conflict with other tests, but it's pretty ugly.